<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primeWindow</class>
 <widget class="QMainWindow" name="primeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlACloud</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="ownCloudStackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="filePage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="QPushButton" name="dolphinButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="text">
           <string>Open Dolphin</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="configPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="configCheckBox">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Backup kConfig automatically</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Backup kConfig every (day):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>31</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="restoreLastConfigButton">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Restore last configuration</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveConfButton">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Backup configuration now</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="configListView"/>
        </item>
        <item>
         <widget class="QPushButton" name="restoreConfigButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Restore selected configuration</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="contactPage">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QCheckBox" name="contactCheckBox">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Synchronize contacts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="contactButton_2">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>75</height>
               </size>
              </property>
              <property name="text">
               <string>Open Contacts</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="calendarCheckBox">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Synchronize calendar</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="calendarButton">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>75</height>
               </size>
              </property>
              <property name="text">
               <string>Open Calendar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="setPage">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="userNameLabel">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>User name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userNameLineEdit">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="serverLabel">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>URL of cloud server:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="serverLineEdit">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhUrlCharactersOnly</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="portLabel">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Port: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="portSpinBox">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="testConnectionPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Test connections</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>horizontalScrollBar</zorder>
       <zorder>userNameLineEdit</zorder>
       <zorder>serverLineEdit</zorder>
       <zorder>portSpinBox</zorder>
       <zorder>userNameLabel</zorder>
       <zorder>serverLabel</zorder>
       <zorder>portLabel</zorder>
       <zorder>testConnectionPushButton</zorder>
       <zorder>horizontalSpacer_6</zorder>
       <zorder>horizontalSpacer_7</zorder>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Menu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="fileButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="configButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Configuration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="contactButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Contacts/Calendar</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingsButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>27</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOwnCloud"/>
  </widget>
  <action name="actionOwnCloud">
   <property name="icon">
    <iconset>
     <normaloff>images/ownCloud.jpg</normaloff>images/ownCloud.jpg</iconset>
   </property>
   <property name="text">
    <string>own&amp;Cloud</string>
   </property>
   <property name="toolTip">
    <string>New ownCloud configuration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ownCloudStackedWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>horizontalScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ownCloudStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
